libafl/src/mutators/mutations.rs

```diff
@@ -314,7 +311,7 @@ where
             Ok(MutationResult::Skipped)
         } else {
             let byte = state.rand_mut().choose(input.bytes_mut()).unwrap();
-            *byte ^= 1 + state.rand_mut().below(NonZero::new(254).unwrap()) as u8;
+            *byte ^= 1 + state.rand_mut().below(nonzero_lit::usize!(254)) as u8;
```

Comment:
It's basically the same length and we only have 30 usages in the whole code base.
I don't think it's worth it adding a new dependency for that (and adding compile time for the extra macro)
Rust optimizes the new away